src/components/timeline/LayerDiscrete.svelte

```diff
@@ -404,76 +462,108 @@
   }
 
   function drawGroupedMode() {
+    // must clear the canvas before a redraw! otherwise the old ungrouped version can linger around and we draw over that!
+    canvas.getContext('2d')?.clearRect(0, 0, canvas.width, canvas.height);
     if (xScaleView !== null) {
-      const collapsedMode = drawHeight <= ViewConstants.MIN_ROW_HEIGHT;
-      let y = collapsedMode ? 0 : ViewConstants.MIN_ROW_HEIGHT - 1; // pad starting y with the min row height to align with activity tree
-      const expectedRowHeight = rowHeight + activityRowPadding;
-      activityTree.forEach(node => {
-        const newY = drawGroup(node, y, expectedRowHeight, !collapsedMode);
-        if (!collapsedMode) {
+      // expanded cannot possibly be false
+      let y = !expanded ? 0 : ViewConstants.MIN_ROW_HEIGHT - 1; // pad starting y with the min row height to align with activity tree
+      const expectedRowHeight = rowHeight + discreteRowPadding;
+
+      discreteTree.forEach(node => {
+        // TODO: add a gap between activities and external events if both are present together.
```

Comment:
I'd lean towards having no gap between activities and external events when collapsed. Right now it looks like there is a gap. My thought is that if you're adding events and activities to the same row you'd want to see them in the same space when collapsed. Also would be good to keep the row the same small collapsed height for consistency. 
<img width="943" alt="image" src="https://github.com/user-attachments/assets/d21d0c81-3441-4cb4-8116-888379b3234a">

Reply:
But I could be convinced otherwise!

Reply:
I think the reasoning here was just so that they were obviously distinct, but I guess there isn't a totally clear reason to or not to collapse them together. We could ask Lyle?

Reply:
Yeah let's ask Lyle / put this one up for discussion.

Reply:
Per meeting: we will stick to one row on collapse. This is implemented in [change collapsed row behavior to one row](https://github.com/NASA-AMMOS/aerie-ui/commit/1a28aaed074d46a57432497ee5f7a5adb9a0f042).